### PR TITLE
Edit user info closes #13

### LINKED ISSUE #13: As a user, I can manage my general account information
shipping address
billing address
credit cards
basic account info (name, email, password, etc)

### PR DESCRIPTION
**Pull request by nathanows**

### CODE REVIEW

**Comment by nathanows** [comment]
image

**Comment by bhargavisatpathy** [comment]
Awesome!!!

**Comment by nathanows** [comment]
The password on: :create method is so you can edit a user profile without needing to re-enter your password again. So on create it validates that you actually enter a password (it cant be blank) and when you edit your profile, if you leave the password fields blank it leaves your password unchanged

File: app/models/user.rb [hunk 1]
```diff
@@ -16,8 +16,8 @@ class User < ActiveRecord::Base
   validates :email, presence: true, length: { maximum: 255 },
                     format: { with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i, },
                     uniqueness: { case_sensitive: false }
-  validates :password, presence: true
-  validates :password_confirmation, presence: true
+  validates :password, presence: true, on: :create
+  validates :password_confirmation, presence: true, on: :create
   validates :display_name, length: { in: 2..32 }, presence: true,
```
Comment (KristaANelson) [inline app/models/user.rb:21]: cool! Nice to know you can do this

File: app/views/seller/users/index.html.erb [hunk 2]
```diff
@@ -1,44 +1,69 @@
...
+      <table>
+        <tr>
+          <td><strong>Full Name:</strong><br /><%= @user.full_name %></td>
+        </tr>
+        <tr>
+          <td><strong>Display Name:</strong><br /><%= @user.display_name %></td>
+        </tr>
+        <tr>
+          <td><strong>Store Address:</strong><br /><%= "#{request.protocol}#{request.host_with_port}/#{@user.slug}/store" %></td>
+        </tr>
```
Comment (KristaANelson) [inline app/views/seller/users/index.html.erb:17]: :sparkles:

**Comment by KristaANelson** [comment]
Nice!

**Pull request merged**